quantaq/client.py

```diff
@@ -216,12 +217,20 @@ def users(self):
         return self._users
 
     @property
-    def teams(self):
+    def orgs(self):
```

Comment:
Let's switch to Organizations as that's the public-facing name of the feature/tool